pkg/api/volumes.go

```diff
@@ -61,11 +61,20 @@ func UpdateVolumeObject(target *common.VolumeObject, source *client.VolumesResou
 }
 
 // CreateVolume : creates a volume with the given name, capacity in the given pool
-func (client *Client) CreateVolume(name, size, pool, poolType string) (*common.VolumeObject, *common.ResponseStatus, error) {
+func (myclient *Client) CreateVolume(name, size, pool, poolType string) (*common.VolumeObject, *common.ResponseStatus, error) {
```

Comment:
The API client shouldn't have to specify the pool type--it's clunky and it will make the client less portable, or force the a portable API client to do work that should be handled by the library.

As a client developer, I would want a simpler CreateVolume(name, size, pool string) function (which doesn't pass the tier argument and works on any system, Linear or Virtual) and if needed, a separate "CreateVolumeInTier(name, size, pool, tier string) which works on virtual systems with performance tier enabled.